src/wp-includes/blocks.php

```diff
@@ -786,7 +787,62 @@ function get_hooked_block_markup( &$anchor_block, $hooked_block_type ) {
 	// However, its presence does not affect the frontend.
 	$anchor_block['attrs']['metadata']['ignoredHookedBlocks'][] = $hooked_block_type;
 
-	return get_comment_delimited_block_content( $hooked_block_type, array(), '' );
+	return get_comment_delimited_block_content( $hooked_block_type, $hooked_block['attrs'], '' );
```

Comment:
We're currently not passing inner blocks to `get_comment_delimited_block_content`.
(We're actually not passing any information other than the block type and attributes.)

Reply:
To do so, we'd basically need to

```suggestion
	return serialize_block( $hooked_block );
```

(Strictly speaking, one might consider using `traverse_and_serialize_blocks`, but not only do we not have access to the block visitors, we also might want to steer away from infinite recursions 😬)

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5835/commits/b34e2cf374d4acb128f2e63ed70de3282b9cc6ff.